es-ES/editor/ui/chrome/composer/editorOverlay.dtd

```diff
@@ -438,7 +438,7 @@
 -->
 
 <!ENTITY listprops.accesskey "l">
-<!ENTITY tableFix.label "Fix Table Layout">
+<!ENTITY tableFix.label "Fijar diseño de la tabla">
```

Comment:
POSSIBLE CORRECTION: This has been translated as "Set table layout" . Does Fix here refer to the action of fixing the layout?